libs/protocol-client/src/lib/utils/ipfs.ts

```diff
@@ -0,0 +1,71 @@
+import { Buffer } from 'buffer';
+import { create, IPFSHTTPClient, CID } from 'ipfs-http-client';
+import { ContributionV1SchemaCid } from './constants';
+
+export type MintedContributionSchemaV1 = {
+  schema: `ipfs://Qmc3fKseQ1JyiJiPsmazx23A6twy6YCkTfqDfHUKZDL5xq`;
```

Comment:
For my own understanding -- this schema value is the text that is the `ipfs://` with the `ContributionV1SchemaCid` appended after the // 

Would we want to use the constant there? thinking of if we update the value in the constants but not here if that would cause issues

Reply:
I tried that. But TS doesn't allow you to use value as a type.